packit_service/worker/build/copr_build.py

```diff
@@ -345,6 +345,13 @@ def get_packit_copr_download_urls(self) -> List[str]:
             )
             raise ex
 
+    def get_job_config_index(self):
```

Comment:
```suggestion
    def get_job_config_index(self) -> int:
```
just to be clear it's indeed int :)